freqtrade/optimize/hyperopt.py

```diff
@@ -32,9 +32,13 @@
 TOTAL_TRIES = None
 _CURRENT_TRIES = 0
 
-TOTAL_PROFIT_TO_BEAT = 3
-AVG_PROFIT_TO_BEAT = 0.2
-AVG_DURATION_TO_BEAT = 50
+TOTAL_PROFIT_TO_BEAT = 0
+AVG_PROFIT_TO_BEAT = 0
+AVG_DURATION_TO_BEAT = 100
+
+# this is expexted avg profit * expected trade count
+# for example 3.5%, 1100 trades, EXPECTED_MAX_PROFIT = 3.85
+EXPECTED_MAX_PROFIT = 3.85
```

Comment:
Should this be used in
https://github.com/gcarq/freqtrade/pull/232/files#diff-4c21adf1de7a357821247a6e8d6abba3R110